base/shell/explorer/appbar.cpp

```diff
@@ -12,6 +12,7 @@
 CAppBarManager::CAppBarManager()
     : m_hAppBarDPA(NULL)
 {
+    ZeroMemory(m_ahwndAutoHideBars, sizeof(m_ahwndAutoHideBars));
```

Comment:
C++ question: doing `m_ahwndAutoHideBars{0}` in the constructor initialization list wouldn't work?

Reply:
Fixed.